dendrocat/radiosource.py

```diff
@@ -197,15 +197,11 @@ def to_catalog(self, dendrogram=None):
                 dendrogram = self.to_dendrogram()
 
         cat = pp_catalog(dendrogram.leaves, self.metadata)
-        cat.add_column(Column(length=len(cat), shape=20, dtype=str),
-                       name='_name')
+        strarr=[str('{:.0f}{:04d}'.format(np.round(self.nu.to(u.GHz).value), idx)) for idx in cat['_idx']]  
+        cat.add_column(Column(data=strarr),name='_name') 
```

Comment:
another super minor thing - I'll do this myself - but we try to avoid whitespace at the end of lines
```suggestion
        cat.add_column(Column(data=strarr), name='_name')
```